README.md

```diff
@@ -103,8 +104,14 @@ USE_SERVER
 3. Save the new text file as `network.lic` in the root directory of the cloned repository.
 4. Open the `Dockerfile`, and comment the line `ENV MLM_LICENSE_FILE`
 5. Uncomment the line `ADD network.lic /usr/local/MATLAB/$MATLAB_RELEASE/licenses/`
-6. Run the docker build command without the `--build-arg LICENSE_SERVER=27000@MyServerName` option. Use a command of the form
+6. Open `matlab_installer_input.txt` in a text editor and edit the following sections:
+    - `licensePath` Add the destination path of the network.lic
+7. Run the docker build command without the `--build-arg LICENSE_SERVER=27000@MyServerName` option. Use a command of the form
 ```
-docker build -t matlab:r2020a .
+docker build -t matlab:r2020a --build-arg MATLAB_RELEASE=R2020a .
```

Comment:
Dockerfile does not have the build-arg MATLAB_RELEASE. Only the Dockerfile.R2019b

Reply:
not sure whether you sw it I have fixed your request a few days ago. Not sure whether this means, that I will have to create a new pull request, or whether these changes still run under the same pull request.